Assets/CameraStartAnim.cs

```diff
@@ -0,0 +1,142 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class CameraStartAnim : MonoBehaviour {
+	public bool done = false;
+	List<Transform> allTargets = new List<Transform>();
+	int zoom_step = 0;
+	//public Transform[] targets;
+	List<Transform> targets = new List<Transform>();
+	float boundingBoxPadding = 0.1f;
+	float minCamera = 8f;
+	float xAngle = 1.012291f; // angle in rad
+	Camera camera;
+	
+	void Awake () 
+	{
+		camera = GetComponent<Camera>();
+		allTargets.Clear ();
+		GameObject[] objs = GameObject.FindGameObjectsWithTag("Player");
+		foreach (GameObject obj in objs) {
+			allTargets.Add(obj.transform);
+		}
+		Invoke ("zoom_in", 0.1f);
+		Invoke ("zoom_out", 2);
+		Invoke ("zoom_in", 3);
+		Invoke ("zoom_out", 5);
+		Invoke ("zoom_in", 6);
+		Invoke ("zoom_out", 8);
+		Invoke ("zoom_in", 9);
+		Invoke ("zoom_done", 11);
+
+	}
+
+	void zoom_in(){
```

Comment:
Function Name.
Please follow Pascal Casing.
Call ZoomIn.
Also for the following functions.